addons/mail/static/src/discuss/core/common/channel_member_list.js

```diff
@@ -39,6 +39,9 @@ export class ChannelMemberList extends Component {
         if (member.persona.type === "guest") {
             return false;
         }
+        if (member.thread.hasNotSelfAsMember) {
+            return false;
+        }
```

Comment:
This doesn't look like a useful restriction, as opening a chat doesn't concern the current channel whatsoever.